.eslintignore

```diff
@@ -1,7 +1,7 @@
 # don't ever lint node_modules
 node_modules
 .github
-data
+/data
```

Comment:
I'm not sure this has changed anything. Deliberately putting a bad ts file in data, then running `./node_modules/.bin/eslint --ext .ts ./` does not show with either `data` or `/data`. Removing the lint from `.eslintignore` then shows the errors.

Reply:
Without the change all folders of name `data` are targeted but but we recently introduced the `integration-tests/data` folder and that has `.ts` files in there. So `/data` just ignores the root `data` folder. Can you confirm that is what is happening?

Reply:
Ah, yes. Now I see what you were fixing. Yeah, worked as you expected.